### PR TITLE
Update cisco-webex-support.md

### PR DESCRIPTION
**Pull request by christiaanbrinkhoff**

### CODE REVIEW

**Comment by christiaanbrinkhoff** [comment]
Windows 365 app release documentation changes.

File: windows-365/enterprise/cisco-webex-support.md [hunk 1]
```diff
@@ -42,7 +42,8 @@ To optimize Cisco Webex, you’ll need to:
 
 ## Requirements
 
-- **Windows Remote Desktop Client**: These instructions don't support connections through a web browser.
+- **Windows 365 app for Windows**: These instructions don't support connections through a web browser.
+- - **Windows Remote Desktop Client**: These instructions don't support connections through a web browser.
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/cisco-webex-support.md:46]:
```suggestion
- **Windows Remote Desktop Client**: These instructions don't support connections through a web browser.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @christiaanbrinkhoff, kindly commit the minor suggested change. Many thanks.

**Comment by ErikjeMS** [comment]
This seems to be a duplicate to PR 3208. Closing.

**Pull request closed without merging**